module_1/first-react-app/src/App.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import SearchBox from './components/SearchBox.js';
+import MovieGenre from './components/MovieGenre.js';
+import CounterLegacy from './components/CounterLegacy.js';
+
+
+
+
+
+function App() {
+
+  useEffect(() => {
+    // Initialize Materialize collapsible when the component mounts
+    const collapsibleElems = document.querySelectorAll('.collapsible');
+    window.M.Collapsible.init(collapsibleElems, {
+      accordion: false
+    });
+
+  }, []); // Empty dependency array ensures that this effect runs once after the initial render
+
+
+  const genres = ['All', 'Action', 'Adventure', 'Comedy', 'Drama', 'Sci-Fi'];
+  const [selectedGenre, setSelectedGenre] = useState('All');
+
+
+  const handleGenreSelect = (genre) => {
+    setSelectedGenre(genre);
+    // Perform any other actions when a genre is selected
+    //alert('Genre is: ' + genre);
+  };
+
+  const handleSearch = (query) => {
+    alert("Searching movies with name: " + query);
+  }
+
+
+
+
```

Comment:
Please remove empty lines

Reply:
Have removed in commit: c7a4e1e3bff35922c071c59fe3905a3da37fefef